src/app/App.tsx

```diff
@@ -10,6 +10,20 @@ import history from './History';
 import { PersistGate } from 'redux-persist/lib/integration/react';
 import * as Visibility from 'visibilityjs';
 
+/**
+ * Use the Patternfly RCUE productized css styles if set by the environment
+ * @example 'env REACT_APP_RCUE=true yarn start'
+ */
+const loadRcueCssIfNeeded = async (): Promise<void> => {
+  // get the flag from command line if set: env REACT_APP_RCUE=true yarn start
+  const useRcue = process.env.REACT_APP_RCUE;
```

Comment:
How exactly does this work? It's replaced at config time? I'd imagine `process.env` not to be available on the browser, no?

Reply:
It's compile time (build time) not runtime. So for your productization build use: 
`env REACT_APP_RCUE=true yarn start` for that build.